tests/test_model.py

```diff
@@ -1000,6 +1000,23 @@ def test_different_item_metadata_types_for_hybrid_pretrained_model(implicit_mode
     assert model_2.item_metadata.equal(model_3.item_metadata)
 
 
+def test_item_metadata_with_nulls_hybrid_pretrained_model(implicit_model,
+                                                          movielens_metadata_df,
+                                                          train_val_implicit_data):
+
+    train, val = train_val_implicit_data
+
+    # create copy of metadata and add column of nulls
+    movielens_metadata_df_with_bad_col = movielens_metadata_df.copy()
+    movielens_metadata_df_with_bad_col['bad_column'] = pd.np.NaN
```

Comment:
My preference is to call `np` directly rather than through Pandas, as this is unintentional Pandas behavior: https://github.com/pandas-dev/pandas/issues/30296 

```suggestion
    movielens_metadata_df_with_bad_col['bad_column'] = np.nan
```

Reply:
Done